vpr/src/base/vpr_types.h

```diff
@@ -1205,7 +1233,7 @@ struct t_trace {
  * occ:        The current occupancy of the associated rr node              */
 struct t_rr_node_route_inf {
     int prev_node;
```

Comment:
In the same vein, we should replace various references to nodes by int index with RRNodeId. Although in some cases like this, since RREdgeIds are unique we may even be able to drop storing the previous node (provided it can be looked up efficiently via the RR graph). Probably best to land this, and do the RRNodeId conversion in another PR.

Reply:
Ya, a follow on PR to convert all the inode to RRNodeId would be good.  But smaller PR's -> faster to review,  faster to merge is probably better.